database/challenge_test.go

```diff
@@ -14,41 +16,80 @@ import (
 	"golang.org/x/crypto/ed25519"
 )
 
-// TestChallengeResponse_LoadFromRequest tests the LoadFromRequest method of
+// TestChallengeResponse_LoadFromRequest tests the LoadFromReader method of
 // ChallengeResponse.
 func TestChallengeResponse_LoadFromRequest(t *testing.T) {
```

Comment:
nit: same update, comment too

Reply:
Thanks. GoLand's rename feature could be better.